### PR TITLE
Bug 2087043: Update manifests to release-4.13

### PR DESCRIPTION
**Pull request by Fedosin**
Bump manifests for CAPI components.

Generated by:
make assets

### CODE REVIEW

**Comment by JoelSpeed** [comment]
If we fetch from upstream, we have no control over what it is that we are generating, we also have no ability to update the assets independent of the upstream, are you absolutely sure we want to be picking this from upstream and not our own downstream main branch?

How will this work for older releases, do they need to be set to an old release?

/hold I'd like to understand a bit more about why this is the value it is today before we merge, @Fedosin can you see if you can work out why this has been pinned to a release branch previously

**Comment by Fedosin** [comment]
> If we fetch from upstream, we have no control over what it is that we are generating

Well, it's quite opposite - we define what changes we want to add here. For instance, we change image names, add OpenShift specific label and annotations, and so on. We even have functions exactly for customizations of upstream manifests:
https://github.com/openshift/cluster-capi-operator/blob/main/hack/assets/customizations.go
https://github.com/openshift/cluster-capi-operator/blob/main/hack/assets/providercustomizations.go

The origional goal was to never touch upstream manifests, keep them as-is in operands repositories, and add modifications during asset generation.

**Comment by JoelSpeed** [comment]
/approve cancel

Some discussion about this during the arch call

File: manifests/0000_30_cluster-api_02_crd.infrastructure-gcp.yaml [hunk 1]
```diff
@@ -1283,12 +1289,6 @@ spec:
                           description: Region is the name of the region where the
                             Subnetwork resides.
                           type: string
-                        routeTableId:
-                          description: 'EnableFlowLogs: Whether to enable flow logging
-                            for this subnetwork. If this field is not explicitly set,
-                            it will not appear in get listings. If not set the default
-                            behavior is to disable flow logging.'
-                          type: boolean
```
Comment (JoelSpeed) [inline manifests/0000_30_cluster-api_02_crd.infrastructure-gcp.yaml:1291]: They dropped a field?

Comment (Fedosin) [reply manifests/0000_30_cluster-api_02_crd.infrastructure-gcp.yaml:1291]: Renamed it - https://github.com/kubernetes-sigs/cluster-api-provider-gcp/pull/665

But yeah, it breaks backward compatibility

Comment (JoelSpeed) [reply manifests/0000_30_cluster-api_02_crd.infrastructure-gcp.yaml:1291]: Right, and we are making this change between major releases right? So it should be ok. In the future we would need to  be consistent about upgrade paths for this

Comment (Fedosin) [reply manifests/0000_30_cluster-api_02_crd.infrastructure-gcp.yaml:1291]: Yes, correct.

**Comment by JoelSpeed** [comment]
Seems there's something wrong with the assets check, can you please look into that and let me know what's wrong with it

**Comment by Fedosin** [comment]
Yes, `asset` failure is expected because I manually updated cluster api operator manifests to make it work. Since they are different from what we have in the forked repo, the job fails. It will be fixed when we merge https://github.com/openshift/cluster-api-operator/pull/9

**Comment by Fedosin** [comment]
We can't merge [it](https://github.com/openshift/cluster-api-operator/pull/9) because we need to have correct manifests here first. I think we agreed on updating manifests in semi-manual mode, otherwise those tests won't pass.

**Comment by JoelSpeed** [comment]
/lgtm
/approve

@Fedosin We need to force this through right? The tests won't pass until other repos are updated? Or should the E2E be passing?

**Comment by Fedosin** [comment]
/hold cancel

no, it's all good - all test passed

**Pull request merged**